Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -649,8 +650,7 @@ static ErrorOr<void> read_start_of_scan(AK::SeekableStream& stream, JPEGLoadingC
         current_scan.spectral_selection_end,
         current_scan.successive_approximation);
 
-    // The three values should be fixed for baseline JPEGs utilizing sequential DCT.
-    if (current_scan.spectral_selection_start != 0 || current_scan.spectral_selection_end != 63 || current_scan.successive_approximation != 0) {
+    if (current_scan.spectral_selection_start > 63 || current_scan.spectral_selection_end > 63 || current_scan.successive_approximation != 0) {
```

Comment:
Should this get a `// FIXME: Support progressive jpegs with current_scan.successive_approximation != 0`?